api/routes/monitoring_routes.py

```
@@ -0,0 +1,135 @@
+                        level = args[1].strip()
+                        message = args[2].strip()
+
+                        for p in puffer:
+                            message += " \n " + p.strip()
+
+                        if level not in selected_levels:
+                            continue
+
+                        logs.append({"time": time, "level": level, "message": message})
+    except Exception:
+        logger.exception(f"failed to read logs")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)